go/ql/lib/semmle/go/frameworks/Twirp.qll

```diff
@@ -0,0 +1,175 @@
+/** Provides models of commonly used functions and types in the twirp packages. */
+
+import go
+private import semmle.go.security.RequestForgery
```

Comment:
```suggestion
private import semmle.go.security.RequestForgeryCustomizations
```
Oops, I didn't spot that this was importing a file that contains the definition of a `Configuration`. There's a QL-for-QL check that helpfully flagged it up. Hopefully we will soon change the way that configurations are implemented to get rid of this booby trap.